pkg/vere/main.c

```diff
@@ -628,6 +628,7 @@ _cw_usage(c3_c* bin_c)
     "utilities:\n",
     "  %s cram %.*s              jam state:\n",
     "  %s dock %.*s              copy binary:\n",
+    "  %s eval                      evaluate hoon from stdin:\n",
```

Comment:
Nit: align right hand side with line above

Reply:
It doesn't line up when printing to the main terminal. It was easier to add the spaces than mess with the for loop a dozen or so lines below.

Reply:
Oh, apologies. I see the %.*s where `<pier>` is normally printed and eval is without that.